src/randomizer.ts

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
This file is perfect example for file with helpers. Move it to helpers folder and find more appropriate name for the file.